### PR TITLE
(IAC-394) [AWS] update viya4-iac-aws from k8s 1.19 to 1.21

### PR DESCRIPTION
**Pull request by riragh**
### Test Coverage
- Verified using updated Dockerfile, cluster was deployed correctly using K8s v1.21.7 and Viya4 deployment on that cluster was successful

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -3,7 +3,7 @@ ARG AWS_CLI_VERSION=2.1.29
 FROM hashicorp/terraform:$TERRAFORM_VERSION as terraform
 
 FROM amazon/aws-cli:$AWS_CLI_VERSION
-ARG KUBECTL_VERSION=1.19.9
+ARG KUBECTL_VERSION=1.21.2
```
Comment (thpang) [inline Dockerfile:6]: Given this is in the docker file and any 1.21 version would support the AWS 1.21 version, which happens to be 1.21.2, why would we not try and keep these the same across all cloud providers? Recently we updated Azure to used kubectl 1.21.7 I believe shown [here](https://github.com/sassoftware/viya4-iac-azure/pull/217/files).

File: README.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ The following are also required:
 #### Terraform Requirements:
 
 - [Terraform](https://www.terraform.io/downloads.html) v1.0.0
-- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) - v1.19.9
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) - v1.21.2
```
Comment (thpang) [inline README.md:48]: Sync with comment above.

**Review by thpang — CHANGES_REQUESTED** [review]
Two comments about provider uniformity. The rest of the changes look good.

**Comment by riragh** [comment]
Updated the K8s version from 1.21.2 to 1.21.7 for uniformity. Ready for re-review

**Review by jarpat — APPROVED** [review]
+1
Built an image using the dockerfile (kubectl version 1.21.7) and was able to create a 1.21 EKS cluster using the example tfvar file.

**Review by thpang — APPROVED** [review]

**Pull request merged**